pixart.js

```diff
@@ -1 +1,23 @@
+function changeBrushColor(){
+  var color = document.getElementById("color-field").value;
+  event.preventDefault();
```

Comment:
Where is this `event` coming from? I'm surprised this portion of the code didn't potentially error out on you. Make sure that you have the `event` object passed in as a parameter to this function.